maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -349,15 +375,31 @@ export const LayerControlPanel = memo(
                               alignItems="center"
                               gutterSize="none"
                               direction="row"
+                              justifyContent={'flexStart'}
                             >
+                              <EuiFlexItem
+                                className="layerControlPanel__layerTypeIcon"
+                                grow={false}
+                              >
+                                <EuiIconTip
+                                  size="m"
+                                  type={LAYER_ICON_TYPE_MAP[layer.type]}
+                                  color={
+                                    visibleLayers.find((l) => l.id === layer.id)
+                                      ? 'success'
+                                      : '#DDDDDD'
+                                  }
+                                  content={getLayerTooltipContent(layer)}
+                                  position={'right'}
+                                />
```

Comment:
Why not add a tool tip to list item entry?

Reply:
Yes, updated.